app/indexers/curation_concerns/work_indexer.rb

```diff
@@ -0,0 +1,16 @@
+module CurationConcerns
+  class WorkIndexer < ActiveFedora::IndexingService
+    include IndexesThumbnails
+    self.thumbnail_path_service = Sufia::WorkThumbnailPathService
+    def generate_solr_document
```

Comment:
That is, why don't we create a Sufia::WorkIndexer that inherits from CurationConcerns::WorkIndexer?